subt/tools/ignstate.py

```diff
@@ -240,8 +240,8 @@ def main():
     cv2.imwrite(args.filename+'.png', img)
     print("created:", args.filename+'.png')
     if args.open:
-        import webbrowser
-        webbrowser.open(args.filename+'.png')
+        from . import startfile
```

Comment:
can you use absolute imports?

Reply:
well, yes... but I was working under the impression that we only support running anything as a module and not as a script, ie. as `python -m <module>` and not as `python module.py`. If not, we can never ever use relative imports.